ToDo-Garden/ToDoGardenUI/Sources/ToDoGardenUIConstant/Constant+SearchGardenButton.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Constant+SearchGardenButton.swift
+//
+//
+//  Created by Wood on 4/19/24.
+//
+
+import Foundation
+
+extension Constant.SearchGardenButton {
+  public enum Layout {}
+  public enum ImageView {}
+}
+
+extension Constant.SearchGardenButton.Layout {
+  public static let cornerRadius: CGFloat = 12
+}
+
+extension Constant.SearchGardenButton.ImageView {
+  public static let trailingConstraint: CGFloat = -12.5
+}
```

Comment:
```suggestion
    public enum Layout {}
}

extension Constant.SearchGardenButton.Layout {
    public enum ImageView { }
}

extension Constant.SearchGardenButton.Layout.ImageView {
  public static let trailingMargin: CGFloat = 12.5
}
```

이렇게 되어야 namespace 단계가 맞지 않을까요? 🧐

Reply:
> 이렇게 되어야 namespace 단계가 맞지 않을까요? 🧐

네 확실히 기존보다 더 명확해보입니다 
다음 커밋에서 반영하였습니다.
https://github.com/ToDo-Garden/ToDo-Garden/pull/79/commits/d0b92dbda1aae0d5a6b37ec7d07254423c27f891